### ISSUE TITLE
Fix repository metadata for some packages

### ISSUE THREAD

**Issue by aaronadamsCA** [system-ui/theme-ui#1776]
**Describe the bug**
Some packages, like [@theme-ui/color](https://www.npmjs.com/package/@theme-ui/color), don't include repository metadata on NPM.

**To Reproduce**
Compare these two NPM packages:

- https://www.npmjs.com/package/@theme-ui/preset-base
- https://www.npmjs.com/package/@theme-ui/color

Only the first package lists its repository as https://github.com/system-ui/theme-ui.

**Expected behavior**
All monorepo packages list their repository as https://github.com/system-ui/theme-ui.

Easiest fix is to ensure this fragment appears in every `packages/*/package.json`:

```json
{
  "repository": "system-ui/theme-ui"
}
```

**Additional context**
This makes it more difficult to configure tools like Renovate, which use the repository metadata to identify packages that live in the Theme UI monorepo and upgrade them in lockstep.

There is support in the `package.json` spec for specifying the directory for each package, like so:

```json
{
  "repository": {
    "type": "git",
    "url": "https://github.com/system-ui/theme-ui",
    "directory": "packages/core"
  }
}
```

But I don't see any tooling that uses this much more verbose format, so I'm not sure the complexity adds anything.

Let me know the preferred format and I'm happy to submit a PR, looks like an easy low-risk update.

**Comment by hasparus** [comment]
Hey @aaronadamsCA! Thanks for the issue! I didn't know the concise format before :) Go with the one you prefer.

**Comment by hasparus** [comment]
:rocket: Issue was released in [`v0.9.2-develop.0`](https://github.com/system-ui/theme-ui/releases/tag/v0.9.2-develop.0) :rocket:

**Comment by aaronadamsCA** [comment]
> I didn't know the concise format before :)

Neither did I, I learned it from this repo!

Thanks for merging; I also filed https://github.com/renovatebot/renovate/pull/10281, so Renovate now has built-in support for Theme UI, which will work correctly as soon as v0.9.2 is released.

**Comment by hasparus** [comment]
:rocket: Issue was released in [`v0.10.0`](https://github.com/system-ui/theme-ui/releases/tag/v0.10.0) :rocket:
